index.html

```diff
@@ -20,12 +20,6 @@
 <body onload="run();">
   <h1>unitncal</h1>
   <p><strong>Sincronizza nativamente il tuo calendario con gli orari dell'università di Trento!</strong></p>
-  <p style="color:red;font-size:3em;">
-    Gli sviluppatori di easyroom hanno da un momento all'altro rotto tutti i
-    link dei calendari, quindi per ora i link generati con questo sito non
-    funzioneranno. Prima o poi lo sistemerò, ma potrebbe passare un po' di
-    tempo, essendo occupato con la tesi. Pull Requests benvenute.
-  </p>
   <p>
     La procedura è un po' contorta a causa della complessità di "easy"room,
```

Comment:
Dato che il nome è completamente cambiato, potresti cambiare tutte le occorrenze di "easyroom" in "easyacademy"?

Reply:
con un `git grep room` dovresti riuscire a trovare quelle mancanti (magari è solo questa)

Reply:
Mi ero limitato a cercare con vscode la parola easyroom su tutta la repository. Ora dovrebbe essere aggiornato.